packages/data-format/package.json

```diff
@@ -39,8 +39,8 @@
     "test:watch": "yarn test --watch"
   },
   "dependencies": {
-    "@ethersproject/bignumber": "5.0.15",
```

Comment:
I'll verify the impact of changing this dependency on the size of the package, but it seems using these packages cause conflicts sometimes